pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -399,7 +399,7 @@ func newPrefixWriter(out io.Writer, prefix string) io.Writer {
 
 func (o *MustGatherOptions) waitForPodRunning(pod *corev1.Pod) error {
 	phase := pod.Status.Phase
-	err := wait.PollImmediate(time.Minute, time.Duration(o.Timeout)*time.Second, func() (bool, error) {
+	err := wait.PollImmediate(3*time.Second, time.Duration(o.Timeout)*time.Second, func() (bool, error) {
```

Comment:
from testing, I believe 3 sec is all we need, the cmd does this:
1) wait for init container running (if you look at the must-gather pod at this point it will show `Init:0/1` while init container runs. (this is the 1st 3 second retry - I haven't seen it take longer than 2 retries)
2) gather container logs - this happens in the init container
3) wait for must-gather pod Running - this doesn't start until the init container is done, so 3 seconds is also good for this one - I haven't seen this take longer than 2 retries, either.  Once the init container completes (step 2 above) container turns to Running. 

With this change, we actually cut a couple of unnecessary minutes off of must-gather!

Reply:
oki doke